Makefile

```diff
@@ -8,6 +8,23 @@ BUILD_BUILDER_IMAGE ?= false
 
 export COLLECTOR_VERSION := $(COLLECTOR_TAG)
 
+podman =
```

Comment:
[nit] can we capitalise this so it's distinguishable from the `podman` command?
```suggestion
PODMAN =
```

Reply:
Yep, why not.